### ISSUE TITLE
Cannot resize exception box to read errror message

### ISSUE THREAD

**Issue by iandobbie** [MicronOxford/cockpit#727]

**Comment by carandraug** [comment]
This issue is Mac specific. The window is as large as the message bug the section with the traceback is too small and the window can't be resized.

**Comment by iandobbie** [comment]
I have added the wx.RESIZE_BORDER to the dialog style which at least allows resizing to see the message. branch 727-mac-resize-exception

**Comment by iandobbie** [comment]
Finally fixed properly. As in  windows the mac text sizer ignore newlines. Fix was to just add WXMAC to the relevant if statement so that the mac also explicitly counts the lines

**Comment by carandraug** [comment]
Adding the `wx.RESIZE_BORDER` does make it possible to resize the frame but it removes the rest of the default style flags:

```
>>> format(wx.DEFAULT_DIALOG_STYLE, 'b')
'100000000000000001100000000000'
>>> format(wx.RESIZE_BORDER, 'b')
'1000000'
```

I fixed this by OR the two flags.

**Comment by iandobbie** [comment]
Oh right. As no other flags were explicitly set I assumed it was ok. Thanks for fixing it.

I actually pulled a further fix which was much better today. But I decided to leave the resize border in case of issues where it is needed

Ian

Sent from my iPhone

On 31 Mar 2021, at 19:03, David Miguel Susano Pinto ***@***.***> wrote:

﻿

Adding the wx.RESIZE_BORDER does make it possible to resize the frame but it removes the rest of the default style flags:

>>> format(wx.DEFAULT_DIALOG_STYLE, 'b')
'100000000000000001100000000000'
>>> format(wx.RESIZE_BORDER, 'b')
'1000000'


I fixed this by OR the two flags.
